### ISSUE TITLE
Chrome 116

### ISSUE THREAD

**Issue by bruhcephalus** [ultrafunkamsterdam/undetected-chromedriver#1338]
Currently trying to use this with a Chromium 116 binary on Linux by setting `version_main=116`.

Error message: `urllib.error.HTTPError: HTTP Error 404: Not Found`.

I know 114 is supported, but setting `version_main=114`  is not an option right now because I only have Chromium 116 installed. (Google makes it pretty impossible to find the damn links to older versions of Chromium...)

**Comment by lonewolf-github** [comment]
This might be of help for getting old versions: https://www.chromium.org/getting-involved/download-chromium/

**Comment by bruhcephalus** [comment]
Thanks for the link, but I should have mentioned that I'd already tried what they describe in that doc. On https://omahaproxy.appspot.com, I can see that the version I want is currently listed on the third row: `linux stable 114.0.5735.106`.

But I've tried all possible combinations of values under all possible `*linux*` directories on https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html without success. All versions I get are from 2022 and before, but I know 114 was released just a few weeks ago.
